src/Language/PureScript/Docs/RenderedCode/RenderType.hs

```diff
@@ -154,12 +154,6 @@ insertPlaceholders :: RenderTypeOptions -> Type -> Type
 insertPlaceholders opts =
   everywhereOnTypesTopDown convertForAlls . everywhereOnTypes (convert opts)
 
-dePrim :: Type -> Type
-dePrim ty@(TypeConstructor (Qualified _ name))
-  | ty == tyBoolean || ty == tyNumber || ty == tyString =
-    TypeConstructor $ Qualified Nothing name
-dePrim other = other
```

Comment:
It is handled inside Pursuit, yeah. Really this function should never have been there at all, it was an oversight to include it. We want to keep this information in because we use it to generate links; see for example https://pursuit.purescript.org/packages/purescript-react/2.0.0/docs/React#t:KeyboardEvent - some of those are broken and I am fairly sure it is because of this function.